README.md

```diff
@@ -37,23 +29,26 @@ The first step is to compile LLVM on your machine
      cmake ..
      make
 
+To run tests, you need to build `lli`.
+
+Or install the LLVM packages on your system:
+--------------------------------------------
+
+On macOS, use pkgsrc and install the llvm package. On CentOS, install the llvm-devel package. On Debian and derivatives, install the llvm-dev package.
+
+It is recommended to install the clang package too.
+
```

Comment:
Please move this before the building LLVM section. I cannot imagine anyone being interested in the building of LLVM from scratch.

Reply:
Ok, changed now.

Reply:
Thanks!